src/main/java/uk/gov/hmcts/ccd/domain/service/common/SecurityClassificationService.java

```diff
@@ -156,12 +156,20 @@ private void filterCollection(SecurityClassification userClassification, Iterato
                                        relevantDataClassificationValue,
                                        userClassification);
                 } else {
+                    LOG.warn("Invalid security classification structure for collection item: {}", relevantDataClassificationValue.toString());
```

Comment:
can we have a collection of collection here?

Reply:
Do we need test for this one?

Reply:
Collection of collections is not possible. It would need to be a collection of complex object containing a collection. So there would be a Complex in the middle. But that's a good point, we need to make sure we test with that scenario.

Reply:
I'll add a test for it, just to be safe 👍